### ISSUE TITLE
Batch Normalization Convolution Layer minor issue

### ISSUE THREAD

**Issue by rdeepmath91** [udacity/deep-learning#198]
The issue is part of the following notebook https://github.com/udacity/deep-learning/blob/master/batch-norm/Batch_Normalization_Solutions.ipynb 
```
"""
DO NOT MODIFY THIS CELL
"""
def conv_layer(prev_layer, layer_depth):
    """
    Create a convolutional layer with the given layer as input.
    
    :param prev_layer: Tensor
        The Tensor that acts as input into this layer
    :param layer_depth: int
        We'll set the strides and number of feature maps based on the layer's depth in the network.
        This is *not* a good way to make a CNN, but it helps us create this example with very little code.
    :returns Tensor
        A new convolutional layer
    """
    strides = 2 if layer_depth % 3 == 0 else 1
    conv_layer = tf.layers.conv2d(prev_layer, layer_depth*4, 3, strides, 'same', activation=tf.nn.relu)
    return conv_layer
```

Above is taken from the 3rd cell, and it's somewhat inconsistent with prior explanation which mentioned `we're always using a 3x3 kernel, ReLU activation functions, strides of 1x1 on layers with odd depths, and strides of 2x2 on layers with even depths`.

Should it be  `strides = 2 if layer_depth % 2 == 0` else 1 instead? If this should be corrected, it affects the other convolution layer (cell 6 and 9) similarly.

**Comment by abhiojha8** [comment]
Though we are a little late to respond, we encourage such _user-specific_ issues to be raised on the Knowledge hub, so that our experienced mentors can chime-in and new students can also learn from the discussion thread. 
Thank you
